tasks/src/image-to-image/data.ts

```diff
@@ -0,0 +1,62 @@
+import { type TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "Synthetic dataset, for image relighting",
+			id: "VIDIT"
+		},
+		{
+			description: "Multiple images of celebrities, used for facial expression translation"
+			id: "CelebA"
```

Comment:
You can use this one from the Hub
```suggestion
			id: "huggan/CelebA-faces"
```